app/view_models/admin/auctions_index_view_model.rb

```diff
@@ -1,6 +1,12 @@
 class Admin::AuctionsIndexViewModel < Admin::BaseViewModel
-  def auctions
-    Auction.all.order(started_at: :desc).map do |auction|
+  attr_reader :auctions
+
+  def initialize(auctions: Auction.all.order(started_at: :desc))
```

Comment:
I wonder if we should put that default argument into a method like `default_auctions` so the args will look like `auctions: default_auctions`